diary.h

```diff
@@ -17,7 +17,7 @@
 #include <locale.h>
 #include <langinfo.h>
 
-#define DIARY_VERSION "0.4"
+#define DIARY_VERSION "0.5"
```

Comment:
```suggestion
#define DIARY_VERSION "0.5-unstable"
```

: )

It could be a bit confusing to have a version number that isn't necessarily true pop up on `--version`.

This way the release 0.5 commit can be tagged v0.5 and remove the -unstable